packages/statemanager/src/ethersStateManager.ts

```diff
@@ -189,9 +189,9 @@ export class EthersStateManager implements StateManager {
     const proof = await this.provider.send('eth_getProof', [address.toString(), [], this.blockTag])
 
     const proofBuf = proof.accountProof.map((proofNode: string) => toBytes(proofNode))
-
-    const trie = new Trie({ useKeyHashing: true })
-    const verified = await trie.verifyProof(keccak256(proofBuf[0]), address.bytes, proofBuf)
+    const veritrie = await Trie.fromProof(keccak256(proofBuf[0]), proofBuf)
```

Comment:
Ah, that's a nice new API! 🙂 👍 

(not sure if final but at least going into a good direction, always looked wrong/out-of-place to me with the explicit Trie instantiation)